__tests__/fixtures/add/install-strict-all/.yarnrc

```diff
@@ -0,0 +1,2 @@
+yarn-offline-mirror "./mirror-for-offline"
```

Comment:
Offline mirror is for not using the internet to install packages during test AFAIK.